hack/dockerfile/binaries-commits

```diff
@@ -1,23 +1,15 @@
 #!/bin/sh
 
-# When updating TOMLV_COMMIT, consider updating github.com/BurntSushi/toml
-# in vendor.conf accordingly
-TOMLV_COMMIT=a368813c5e648fee92e5f6c30e3944ff9d5e8895
-
-# When updating RUNC_COMMIT, also update runc in vendor.conf accordingly
-RUNC_COMMIT=6c55f98695e902427906eed2c799e566e3d3dfb5
-
-# containerd is also pinned in vendor.conf. When updating the binary
-# version you may also need to update the vendor version to pick up bug
-# fixes or new APIs.
-CONTAINERD_COMMIT=cfd04396dc68220d1cecbe686a6cc3aa5ce3667c # v1.0.2
-TINI_COMMIT=949e6facb77383876aeff8a6944dde66b3089574
-
-# LIBNETWORK_COMMIT is used to build the docker-userland-proxy binary. When
-# updating the binary version, consider updating github.com/docker/libnetwork
-# in vendor.conf accordingly
-LIBNETWORK_COMMIT=20dd462e0a0e883437a274bd61df4bc4de980830
-VNDR_COMMIT=a6e196d8b4b0cbbdc29aebdb20c59ac6926bb384
-
-# Linting
-GOMETALINTER_COMMIT=bfcc1d6942136fd86eb6f1a6fb328de8398fbd80
+# This file used to contain a list of env vars with git commit sha's for building
```

Comment:
Is this still used somewhere? Could we remove it?

Reply:
I'll test... I think I originally removed it and then added it back in because something broke... been awhile now.

Reply:
It's used in the go autogen stuff.

Reply:
Could autogen source the files it needs directly?